gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-fence-delete-client.groovy

```diff
@@ -44,10 +44,10 @@ pipeline {
                             "implicit-test-client"
                             "basic-test-abc-client"
                             "jenkinsClientTester"
```

Comment:
let's change to snake case here, we will have to add a new client jenkins-client-tester in user.yaml to avoid breaking existing pipeline